tests/test_core.py

```diff
@@ -262,3 +262,23 @@ def test_complex_struct(self):
         self.db['test'] = test_value
         assert self.db['test'] == test_value
         assert self.get_json('test') == json.dumps(test_value)
+
+
+class TablenamesTest(TestCaseBackport):
+
+    def test_tablenames(self):
+        fname = norm_file('tests/db/tablenames-test-1.sqlite')
+        db = sqlitedict.SqliteDict(fname) 
```

Comment:
what's about an empty file?

Reply:
not sure what you mean. in any case table `unamed` will be created

Reply:
@transfluxus aha, OK!